tvm_cli/src/completion.rs

```diff
@@ -68,13 +68,11 @@ fn print_paths(prefix: &str) {
         return;
```

Comment:
let Ok(paths) = std::fs::read_dir(folder) else {
    return;
};
...
for path in paths.flatten() {